spacepy/toolbox/__init__.py

```diff
@@ -1520,12 +1534,12 @@ def windowMean(data, time=[], winsize=0, overlap=0, st_time=None, op=np.mean):
     Windowing mean function, window overlap is user defined
 
     Parameters
-    ==========
+    ----------
     data : array_like
         1D series of points
     time : list (optional)
         series of timestamps, optional (format as numeric or datetime)
-        For non-overlapping windows set overlap to zero.
+        For non-overlapping windows set overlap to zero. Must be same length as data.
```

Comment:
Too long again.

Reply:
Fixed, I note there are dozens of too ling lines that were not touched in this PR.